automation/src/test/scala/org/broadinstitute/dsde/test/api/BillingProjectOwnerPermissionSpec.scala

```diff
@@ -0,0 +1,260 @@
+
+            Rawls.billing.removeUserFromBillingProject(projectName, owner.email, BillingProjectRole.Owner)
+            AuthDomainMatcher.checkNotVisibleNotAccessible(projectName, workspaceName)(owner.makeAuthToken())
+          }
+        }
+      }
+    }
+
+    "+ group member, create workspace, + project owner, - group member" in {
+
+      val creator = UserPool.chooseProjectOwner
+      val owner = UserPool.chooseStudent
+      implicit val token: AuthToken = creator.makeAuthToken()
+
+      withCleanBillingProject(creator) { projectName =>
+        withGroup("group", List(owner.email)) { groupName =>
+          withCleanUp {
+            withWorkspace(projectName, "GroupApiSpec_workspace", Set(groupName)) { workspaceName =>
+
+              AuthDomainMatcher.checkNotVisibleNotAccessible(projectName, workspaceName)(owner.makeAuthToken())
+
+              Rawls.billing.addUserToBillingProject(projectName, owner.email, BillingProjectRole.Owner)
+              register cleanUp Rawls.billing.removeUserFromBillingProject(projectName, owner.email, BillingProjectRole.Owner)
+              AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(groupName))(owner.makeAuthToken())
+
+              Sam.user.removeUserFromPolicy(groupName, GroupRole.Member.toString, owner.email)
+              AuthDomainMatcher.checkVisibleNotAccessible(projectName, workspaceName)(owner.makeAuthToken())
+            }
+          }
+        }
+      }
+    }
+
+    "+ group member, create workspace, + project owner, - project owner" in {
+
+      val creator = UserPool.chooseProjectOwner
+      val owner = UserPool.chooseStudent
+      implicit val token: AuthToken = creator.makeAuthToken()
+
+      withCleanBillingProject(creator) { projectName =>
+        withGroup("group", List(owner.email)) { groupName =>
+          withCleanUp {
+            withWorkspace(projectName, "GroupApiSpec_workspace", Set(groupName)) { workspaceName =>
+
+              AuthDomainMatcher.checkNotVisibleNotAccessible(projectName, workspaceName)(owner.makeAuthToken())
+
+              Rawls.billing.addUserToBillingProject(projectName, owner.email, BillingProjectRole.Owner)
+              register cleanUp Try(Rawls.billing.removeUserFromBillingProject(projectName, owner.email, BillingProjectRole.Owner)).recover {
+                case _: RestException =>
+              }
+              AuthDomainMatcher.checkVisibleAndAccessible(projectName, workspaceName, List(groupName))(owner.makeAuthToken())
+
+              Rawls.billing.removeUserFromBillingProject(projectName, owner.email, BillingProjectRole.Owner)
+              AuthDomainMatcher.checkNotVisibleNotAccessible(projectName, workspaceName)(owner.makeAuthToken())
+            }
+          }
+        }
+      }
+    }
+
+    "create workspace, + project owner, + group member, - group member" in {
```

Comment:
This test and the three tests below it aren't actually doing what they state. When the workspace is created, the `owner` user should be neither a billing project owner nor an auth domain member. Those permissions should then be added via api calls in the order specified by the test. Currently, the `owner` user is already a billing project owner or auth domain member when the workspace is created, which means these tests are the same as previous tests in this suite.